{"time":"2023-12-03 14:46:49","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-03 14:47:12","level":"INFO","module":"__init__","method":"setup","line":479,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Normal setup"}
{"time":"2023-12-03 19:58:58","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-06 00:07:48","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-06 18:36:57","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-06 22:15:48","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-07 22:20:22","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-07 22:22:27","level":"INFO","module":"__init__","method":"setup","line":463,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Initial setup"}
{"time":"2023-12-07 22:33:41","level":"ERROR","module":"__init__","method":"_conda_output_ready","line":1038,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Multiple Errors Encountered."}
{"time":"2023-12-07 22:33:45","level":"INFO","module":"__init__","method":"setup","line":479,"path":"\/Users\/mereyazhibekova\/anaconda3\/lib\/python3.11\/site-packages\/anaconda_navigator\/widgets\/main_window\/__init__.py","message":"Normal setup"}
